Assets/Scripts/Interaction/PlayerInteractor.cs

```diff
@@ -21,22 +22,50 @@ private void Start()
         {
             if (!isLocalPlayer) Destroy(this);
             hands = GetComponent<Hands>();
-            if (!mainCamera) mainCamera = Camera.main;
+            mainCamera = Camera.main;
         }
 
         protected virtual void Update()
         {
             if (!mainCamera) return;
+            if (!Input.GetButtonDown("Click")) return;
```

Comment:
While I am for less indentation I think it lessens the readability. The log is "if you click, do something", but this says "if you don't click, don't do anything"

Reply:
! operations are perfectly reasonable and readable. This isn't a double negative or anything confusing.

"if you're not clicking, skip".